test/Driver/response-file.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: echo "-DTEST0" > %t.0.resp
-// RUN: %target-build-swift -typecheck @%t.0.resp %s 2>&1 | %FileCheck %s -check-prefix=SHORT
+// RUN: %target-build-swift -typecheck %s @%/t.0.resp 2>&1 | %FileCheck %s -check-prefix=SHORT
```

Comment:
I don't think this one would matter?

Reply:
Yup, it doesn't. I'll remove it. It's broken because response files don't work on windows right now, but because of the path.